sofa-dashboard-backend/pom.xml

```diff
@@ -27,7 +27,7 @@
         <main.user.dir>../</main.user.dir>
 
         <!-- Dependencies about -->
-        <mysql.connector.version>5.1.35</mysql.connector.version>
+        <mysql.connector.version>8.0.26</mysql.connector.version>
```

Comment:
@OrezzerO you should provide a diff of mysql 5.x & mysql 8.0 , and why we need to upgrade this storeage type version. mysql 8.0 should only be available as an option.

in addition, when you upgrade mysql version, whether you need to consider changing the configuration，such as the Driver.
```
com.mysql.cj.jdbc.Driver
```